### ISSUE TITLE
Support other reporting policies.

### ISSUE THREAD

**Issue by argahsuknesib** [pbonte/RSP-JS#2]

**Comment by pbonte** [comment]
Indeed, I did not add all of them yet. We did already implement them in RSP4J, so we can look there

**Comment by argahsuknesib** [comment]
Yes, I found this TODO in the code. Removed it from there, and added on the issue so that we can track it better.
